### PR TITLE
LPS-128419 Added missing classes

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
Hey @marcoscv-work, I added some missing classes but it is still not working... this css https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/components/_icons.scss#L22 is not being applied.... 

And this is the markup generated
Sorting button markup

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/FilterOrderControls.js [hunk 1]
```diff
@@ -24,8 +24,14 @@ const FilterOrderControls = ({
 	disabled,
 	filterDropdownItems,
 	onFilterDropdownItemClick,
+	sortingOrder,
 	sortingURL,
 }) => {
+	const sortingClasses =
+		sortingOrder == 'desc'
```
Comment (julien) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/FilterOrderControls.js]: you probably want `===` (I know that won't fix the issue you commented, but I was curious and had a look)

**Comment by julien** [comment]
Hey @marcoscv-work @ambrinchaudhary I was just wondering why we're not sending this to @liferay-frontend, is this just a test?

**Comment by ambrinchaudhary** [comment]
> Hey @marcoscv-work @ambrinchaudhary I was just wondering why we're not sending this to @liferay-frontend, is this just a test?

Hey @julien, just because it is still not working... @marcoscv-work will check what is failing... maybe the svg is wrong, or  check why the styles are not being applied...

**Comment by julien** [comment]
@ambrinchaudhary thanks for making that clear :)

**Comment by jbalsas** [comment]
> Hey @julien, just because it is still not working... @marcoscv-work will check what is failing... maybe the svg is wrong, or check why the styles are not being applied...

It wouldn't hurt doing this directly in @liferay-frontend. That way it's easier for anyone in the team to weigh in on this and help push it forward. The fact that it's still not working shouldn't make a whole lot of difference, and there's the "Draft" feature to prevent accidental forwards if that's a concern.

**Comment by ambrinchaudhary** [comment]
Sent to https://github.com/liferay-frontend/liferay-portal/pull/928

**Pull request closed without merging**